pkg/testcase/support/awsutils.go

```diff
@@ -1,14 +1,16 @@
-package aws
+package support
```

Comment:
do you mind the file be something like: 
- `aws.go`  or `instances.go`.  `ec2s.go` , `infra.go`. something on this path ?

Reply:
@mdrahman-suse

Reply:
we dont need utils in the name, since its already in a support/helper pkg

Reply:
Updated